chapters/research-design.tex

```diff
@@ -1,288 +1,586 @@
-The running variable should not be the outcome of interest,
-and while it can be time, that may require additional modeling assumptions.
-Those who qualify are given the intervention and those who don't are not;
-this process substitutes for explicit randomization.\sidenote{\url{http://blogs.worldbank.org/impactevaluations/regression-discontinuity-porn}}
-
-For example, imagine that there is a strict income cutoff created
-for a program that subsidizes some educational resources.
-Here, income is the running variable.
-The intuition is that the people who are ``barely eligible''
-should not in reality be very different from those who are ``barely ineligible'',
-and that resulting differences between them at measurement
-are therefore due to the intervention or program.\cite{imbens2008regression}
-For the modeling component, the \textbf{bandwidth},
-or the size of the window around the cutoff to use,
-has to be decided and tested against various options for robustness.
-The rest of the model depends largely on the design and execution of the experiment.
-
-%------------------------------------------------
-
-\section{Quasi-experimental designs}
+The classic method is the \textbf{randomized control trial (RCT)}.\sidenote{
+  \url{https://dimewiki.worldbank.org/wiki/Randomized_Control_Trials}}
+  \index{randomized control trials}
+In randomized control trials, the control group is randomized --
+that is, from an eligible population,
+a random subset of units are not given access to the treatment,
+so that they may serve as a counterfactual for those who are.
+A randomized control group, intuitively, is meant to represent
+how things would have turned out for the treated group
+if they had not been treated, and it is particularly effective at doing so
+as evidenced by its broad credibility in fields ranging from clinical medicine to development.
+Therefore RCTs are very popular tools for determining the causal impact
+of specific programs or policy interventions.
+However, there are many types of treatments that are impractical or unethical
+to effectively approach using an experimental strategy,
+and therefore many limitations to accessing ``big questions''
+through RCT approaches.\sidenote{
+  \url{https://www.nber.org/papers/w14690.pdf}}
+
+Randomized designs all share several major statistical concerns.
+The first is the fact that it is always possible to select a control group,
+by chance, which is not in fact very similar to the treatment group.
+This feature is called randomization noise, and all RCTs share the need to assess
+how randomization noise may impact the estimates that are obtained.
+Second, takeup and implementation fidelity are extremely important,
+since programs will by definition have no effect
+if the population intended to be treated
+does not accept or does not receive the treatment.
+Unfortunately, the loss of power happens with relatively small implementation gaps and is highly nonlinear:
```

Comment:
```suggestion
Unfortunately, the loss of statistical power that happens with relatively small implementation gaps is highly nonlinear:
```

Reply:
Slight loss of meaning as suggested. Revised for clarity: e704e2ae2784773c6a89b52a376664706dbe7950